css/style.css

```diff
@@ -0,0 +1,206 @@
+:root {
+  --baseGray-40: #b5bec6;
+  --baseWhite: #ffffff;
+  --baseGray-80: #4a5660;
+  --date-hover: #cbbcbc;
+}
+
+[data-theme="dark"] {
+  --baseGray-80: #ffffff;
+  --baseWhite: #464646;
+  --date-hover: #363232;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100%;
+  width: 100%;
+  background-color: var(--baseWhite);
+}
+
+.app {
+  height: 500px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 0 auto;
+  border-radius: 8px;
+  justify-content: center;
+}
+
+.calendar {
+  background-color: var(--baseWhite);
+  display: inline-flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 22px;
+  padding: 24px;
+  border-radius: 8px;
+  overflow: hidden;
+  box-shadow: 0 0 20px 1px rgba(0, 0, 0, 0.1);
+}
+
+.display-month-year {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  align-self: stretch;
+}
+
+.icons {
+  width: 16px;
+  height: 16px;
+  border: none;
+  background-color: var(--baseWhite);
+  cursor: pointer;
+}
+
+.month-year-name {
+  width: fit-content;
+  margin-top: -1px;
+  font-family: "AvenirNext LT Pro-Demi", Helvetica;
+  font-weight: 400;
+  color: var(--baseGray-80);
+  font-size: 14px;
+  text-align: center;
+  letter-spacing: 0;
+  line-height: 14px;
+  white-space: nowrap;
+  font-style: normal;
+}
+
+.weekday-title {
+  display: flex;
+  align-items: flex-start;
+  gap: 8px;
+  align-self: stretch;
+}
+
+.weekday-title-name {
+  display: flex;
+  width: 30px;
+  height: 20px;
+  align-items: center;
+  justify-content: center;
+  gap: 10px;
+  color: var(--baseGray-40);
+  font-family: "AvenirNext LT Pro-Demi", Helvetica;
+  font-weight: 400;
+  font-size: 10px;
+  letter-spacing: 1.5px;
+  line-height: 12px;
+}
+
+.days {
+  display: grid;
+  grid-template-columns: repeat(7, 1fr);
+  grid-template-rows: repeat(6, 1fr);
+  grid-gap: 8px;
+  flex: 0 0 auto;
+}
+
+.days li {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 30px;
+  height: 30px;
+  background-color: var(--baseWhite);
+  border-radius: 50%;
+  font-family: "AvenirNext LT Pro-Demi", Helvetica;
```

Comment:
@raaj-oo7 Why is font family declared multiple times in css?
Just declare it in body and it will be inherited to all the elements